core/controllers/editor_test.py

```diff
@@ -554,8 +554,13 @@ def test_exploration_download_handler_for_default_exploration(self):
 
         # Check download to JSON.
         exploration.update_objective('Test JSON download')
```

Comment:
Do we still need this line? (Also check whether we need the exploration variable?)

Reply:
> Do we still need this line? (Also check whether we need the exploration variable?)

we are using exploration.id without we cannot do anything

Reply:
we have `exp_id`, right?

Reply:
okay but it wouldn't make a difference right ? just a better practice

Reply:
@DubeySandeep Thanks for the help this is my first PR in an open-source project which is outside android domain

Reply:
> okay but it wouldn't make a difference right ? just a better practice

It's not good to define a variable that is not used. (Remove the line 521 as it's defining exploration variable which isn't used anywhere (and also making unwanted DB calls.))

Reply:
> @DubeySandeep Thanks for the help this is my first PR in an open-source project which is outside android domain

You are always welcome and congrats on your first PR to Oppiia! :)